source/documentation/using_the_api/getting_updates.md

```diff
@@ -1,42 +1,111 @@
-## Updating registers
+## Getting updates
 
-You can find the latest entry number by looking at the register information (use the [`GET /register` endpoint](#get-register)) and comparing the most recent entry number with your own copy.
+This guide assumes you know the current highest entry number for your copy of a register. 
 
-Example request: `https://local-authority-eng.register.gov.uk/register`
+You can regularly compare this value with new queries to a register to see if there are any updates.
 
-Download a full new copy of the register by using [the `GET /download-register` endpoint](#get-download-register).
+### Return all new entries since your last update
 
-Download all updates for one record according to a particular key by [using the `GET /records/{key}/entries` endpoint](#get-records-key-entries):
+You can use the [`GET /entries/` endpoint](#getentries) with the `start` and
+`limit` parameters to return specific entries to a register. 
 
-Example URL: `https://local-authority-eng.register.gov.uk/records/KIN/entries`
+For example, your last highest entry number from the `country` register could
+be 205. You could then use the following request to find out if there are any
+new entries:
 
-Example request: `curl --request GET --url https://local-authority-eng.register.gov.uk/records/KIN/entries --header 'Accept: application/json' --header 'Authorization: YOUR-API-KEY-HERE'`
+```http
+GET /entries/?start=206 HTTP/1.1
+Host: country.register.gov.uk
+Accept: application/json
+Authorization: YOUR-API-KEY-HERE
+```
 
 Example response:
 
-```
+```http
+HTTP/1.1 200
+Content-Type: application/json
+Link <?start=206&limit=100>; rel="previous"
```

Comment:
This should be `106` instead of `206`:

```
Link <?start=106&limit=100>; rel="previous"
```

Reply:
Also, would it be more complete to show a example where there is a "next" link so we can reinforce the idea of consuming pages up until there are no "next" links left?